platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -268,6 +269,36 @@ beforeSettings {
 Calling link:{javadocPath}/org/gradle/caching/local/DirectoryBuildCache.html#setRemoveUnusedEntriesAfterDays-int-[buildCache.local.removeUnusedEntriesAfterDays] is deprecated and this method will be removed in Gradle 9.0.
 If set to a non-default value, this deprecated setting will take precedence over `Settings.caches.buildCache.setRemoveUnusedEntriesAfterDays()`.
 
+[[gradle_enterprise_extension_deprecated]]
+==== Deprecated Kotlin DSL gradle-enterprise plugin block extension ====
+
+In `settings.gradle.kts` (Kotlin DSL), you can use `gradle-enterprise` in the plugins block to apply the Gradle Enterprise plugin with the same version as `gradle --scan`.
+```kotlin
+plugins {
+    `gradle-enterprise`
+}
+```
+
+There is no equivalent to this in `settings.gradle` (Groovy DSL).
+
+Gradle Enterprise has been renamed to Develocity and the `com.gradle.enterprise` plugin has been renamed to `com.gradle.develocity`.
+Therefore, the `gradle-enterprise` plugin block extension has been deprecated and will be removed in Gradle 9.0.
+Instead of introducing a `develocity` plugin block extension for the Kotlin DSL, you should use the `com.gradle.develocity` plugin id directly.
```

Comment:
💭 This would work better as a separate paragraph.

I think the wording needs to be improved, because currently it is "Instead of (us) introducing, you should ..."

Maybe:
```
The Develocity plugin must be applied with an explicit plugin id and version. The `develocity` shorthand is not available in the plugins block.
```

Reply:
I adjusted the text, PTAL!